mcc/odin/odin/snapshot_list.go

```diff
@@ -18,6 +20,14 @@ func ListSnapshots(
 	if err != nil {
 		return
 	}
+	sort.Slice(
+		output.DBSnapshots,
+		func(i, j int) bool {
+			return output.DBSnapshots[i].SnapshotCreateTime.After(
+				*output.DBSnapshots[j].SnapshotCreateTime,
+			)
+		},
```

Comment:
Maybe my last comment was not very clear...
`sort.Slice` uses the function as meaning less, so element `i` should be before, in the list, than element `j`.
So, the time comparison used should be `Before` to be chronologically correct, but since this is wanted to order from newer to older, this is using `After`